### PR TITLE
Update pubspec.yaml

### PR DESCRIPTION
**Pull request by devoncarew**
- `2.0.0-dev.infinity`

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,4 +9,4 @@ description: >
   This library declares readable symbolic names for character codes.
 homepage: https://github.com/dart-lang/charcode
 environment:
-  sdk: '>=1.0.0 <2.0.0'
```
Comment (kevmoo) [inline pubspec.yaml]: Mark the version 1.1.2-dev then?

File: pubspec.yaml [hunk 2]
```diff
@@ -9,4 +9,4 @@ description: >
...
+  sdk: '>=1.0.0 <2.0.0-dev.infinity'
```
Comment (lrhn) [inline pubspec.yaml]: Consider just removing the constraint entirely.
If we change the language enough to break `const int id = integerLiteral;`, then we have bigger problems than this package being considered compatible.

Comment (devoncarew) [inline pubspec.yaml]: `<2.0.0-dev.infinity` matches what we're doing with other packages. `<=3.0.0` may make sense for this one; I don't have a strong opinion here -

Comment (kevmoo) [inline pubspec.yaml]: Don't rock the boat. I hear the comment from @lrhn , but I'd rather not have to special case packages.

File: pubspec.yaml (hunk 1 above)
Comment (devoncarew) [inline pubspec.yaml]: done

File: pubspec.yaml (hunk 2 above)
Comment (devoncarew) [inline pubspec.yaml]: `<2.0.0-dev.infinity` it is -

**Comment by devoncarew** [comment]
updated; ptal

File: pubspec.yaml [hunk 3]
```diff
@@ -9,4 +9,4 @@ description: >
   This library declares readable symbolic names for character codes.
 homepage: https://github.com/dart-lang/charcode
 environment:
-  sdk: '>=1.0.0 <2.0.0-dev.infinity'
+  sdk: '>=1.0.0 <=2.0.0-dev.infinity'
```
Comment (kevmoo) [inline pubspec.yaml]: Huh? `=`? We'll never ship a 2.0.0-dev.infinity version...

**Pull request merged**